src/main/java/com/google/graphgeckos/dashboard/storage/ParsedBuildbotData.java

```diff
@@ -59,7 +59,7 @@ public ParsedBuildbotData(String commitHash, String builderName,
    * @return true if all required fields are available, false if not.
    */
   boolean validUpdateData() {
-    return (commitHash != null && builderName != null && logs != null && status != null);
```

Comment:
Please keep in mind when merging that both validUpdateData and toBuilder have been deprecated

Reply:
Thanks. Is the deprecation formally communicated in another PR? Should I manually remove the methods? @MagoDelBlocco

Reply:
It has been done in #79. I left this as a heads-up for the merging process.

Reply:
Cool, thanks for the heads-up.